rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_types_CAV.py

```diff
@@ -29,7 +29,10 @@ def are_all_terminal_types_cav(rmi_b, terminal_unit_id_list):
 
     for terminal_b_id in terminal_unit_id_list:
         terminal_b = find_exactly_one_terminal_unit(rmi_b, terminal_b_id)
-        if terminal_b.get("type") != TERMINAL_TYPE.CONSTANT_AIR_VOLUME:
+        if (
```

Comment:
There is one twist here, which took me very long to figure out.
We have to code in this way because are_all_terminal_types_CAV function allows missing `type` parameter as PASS.

`if terminal_b.get("type") != TERMINAL_TYPE.CONSTANT_AIR_VOLUME` will result in False flag if `terminal_b` has no `type`

`if(terminal_b.get("type") is not None...` short-circuit the None result so None will not result in False flag.

Reply:
Of course you are right. These are not equivalent.  Gotcha